### ISSUE TITLE
你好，想问一下，在测试unet_proto模型训练出来的模型时，在test.py中的关于分类器的选择的代码是怎样的。

### ISSUE THREAD

**Issue by weng-hb** [xjiangmed/LTUDA#2]
将你的源代码改为
            elif args.ulp_classifier:
                outputs = net(data_bat)
                logits = outputs[0]
                compact_pred = torch.argmax(logits, dim=1)
                compact_pred = compact_pred.data.cpu().numpy()
                out = compact_pred[0,:,:].copy()
虽然能跑了，但是输出的结果不行。
大佬教教小白吧

**Comment by weng-hb** [comment]
包括使用你给的模型也是这样

**Comment by xjiangmed** [comment]
Sorry, I forgot to update the test code. I will check and re-upload the test code as soon as possible.

**Comment by weng-hb** [comment]
Thank you very much!哥，给你磕了

**Comment by weng-hb** [comment]
呜呜呜，谢谢啦哥！十分感谢！

**Comment by LeviAov** [comment]
> 呜呜呜，谢谢啦哥！十分感谢！

你好，请问你复现成功了吗？这个代码

**Comment by weng-hb** [comment]
> > 呜呜呜，谢谢啦哥！十分感谢！
> 
> 好，请问你复现成功了吗？

可以复现，但是我之前训练和测试用的是两个不同的环境，时间有点久远，大致记不清了

**Comment by weng-hb** [comment]
> > 呜呜呜，谢谢啦哥！十分感谢！
> 
> 好，请问您复现成功了吗？

可以复现，但是我之前复现的时候训练和测试用的是两个环境，过了两个月了，具体细节记不太清了

**Comment by weng-hb** [comment]
> > 呜呜呜，谢谢啦哥！十分感谢！
> 
> 你好，请问你复现成功了吗？这个代码

![自己训练的模型_第二阶段_测试结果](https://github.com/user-attachments/assets/2949b255-ea4a-4d91-81ef-f1e9d94505ee)

**Comment by LeviAov** [comment]
Hi, there was an error after I trained with the instructions you gave me. But you're giving data sets that end in.npz, so why would you need a.NP file?



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "xjiangmed/LTUDA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月5日(星期四) 下午4:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [xjiangmed/LTUDA] 你好，想问一下，在测试unet_proto模型训练出来的模型时，在test.py中的关于分类器的选择的代码是怎样的。 (Issue #2)





 
Closed #2 as completed.
